dxr/plugins/core.py

```diff
@@ -36,6 +35,14 @@
         'trigrams': {
             'type': 'string',
             'analyzer': 'trigramalyzer'
+        },
+        'segments': {
```

Comment:
I don't think we ever use segments or segments_lower in a LINE query (confirm?), so we can save some index space (and maybe indexing CPU) by not including them on LINE docs.